examples/custom_interaction.py

```diff
@@ -0,0 +1,32 @@
+import nextcord
+
+
+class MyInteraction(nextcord.Interaction):
+    async def send_embed(self, *, title: str, description: str):
+        """Easily send a basic embed."""
+        embed: nextcord.Embed = nextcord.Embed(
+            title=title,
+            description=description,
+        )
+        embed.set_author(name=self.user.display_name, icon_url=self.user.display_avatar)
+        await self.send(embed=embed)
```

Comment:
I'd probably make this return the partialinteractionmessage/webhookmessage if I were doing this, but it's just an example, so it's fine either way